worlds/shivers/Rules.py

```diff
@@ -176,19 +195,88 @@ def set_rules(world: "ShiversWorld") -> None:
     multiworld = world.multiworld
     player = world.player
 
-    rules_lookup = get_rules_lookup(player)
+    rules_lookup = get_rules_lookup(world, player)
     # Set required entrance rules
     for entrance_name, rule in rules_lookup["entrances"].items():
         multiworld.get_entrance(entrance_name, player).access_rule = rule
```

Comment:
These can all use the helper/convenience methods instead.
This applies to quite a few places in multiple files
```suggestion
        world.get_entrance(entrance_name).access_rule = rule
```

Reply:
Ah didn't know about the convenience methods, that does clean up a fair bit.